std-bits/table/src/main/java/org/enso/table/expressions/ExpressionVisitorImpl.java

```diff
@@ -224,6 +226,29 @@ public Value visitUnaryMinus(ExpressionParser.UnaryMinusContext ctx) {
     return executeMethod("*", visit(ctx.expr()), Value.asValue(-1));
   }
 
+  @Override
+  public Value visitAtom(ExpressionParser.AtomContext ctx) {
+    var atomName = ctx.IDENTIFIER().getText();
+    return makeConstructor.apply(toCamelCase(atomName));
+  }
+
+  private static String toCamelCase(String name) {
+    var builder = new StringBuilder(name.length());
+    boolean convertNext = true;
+    for (var c : name.toCharArray()) {
+      if (convertNext) {
```

Comment:
`Foo_Bar` will be `fooBar` but
`Foo__Bar` will be `foobar`

Reply:
First letter is capitalized but yes there was a bug with multiple `_`.

Should now map `fOO__baR` to `Foo__Bar`